src/components/calendar/calendar.jsx

```diff
@@ -42,32 +43,146 @@ function getCellClassName(iterator, month, highlightedDate) {
   return styles['not-current-date'];
 }
 
-function getCell(iterator, month, highlightedDate) {
-  const date = iterator.getDate();
-  const className = getCellClassName(iterator, month, highlightedDate);
-
-  iterator.setDate(date + 1);
+function isEndDate(date) {
+  const nextDate = new Date(date.getYear(), date.getMonth(), date.getDate() + 1);
+  return date.getMonth() !== nextDate.getMonth();
+}
 
-  return (
-    <td className={className}>
-      {date}
-    </td>
-  );
+function getDaysInMonth(date, nextMonth) {
+  let month = date.getMonth() + nextMonth;
+  if (isEndDate(date)) month += 1;
+  return new Date(date.getYear(), month, 0).getDate();
 }
 
 function Calendar(props) {
   const highlightedDate = new Date(props.value ? `${props.value}T00:00` : Date.now());
   const [year, setYear] = useState(highlightedDate.getFullYear());
   const [month, setMonth] = useState(highlightedDate.getMonth());
+  const [active, setActive] = useState(false);
+  const [focusedDate, setFocusedDate] = useState(new Date(props.value ? `${props.value}T00:00` : Date.now()));
+  let refs = {};
+
+  const getCell = (iterator, cellMonth, cellHighlightedDate) => {
+    const date = iterator.getDate();
+    const className = getCellClassName(iterator, cellMonth, cellHighlightedDate);
+    const ref = React.createRef();
+
+    const cellRef = {};
+    cellRef[iterator.toDateString()] = ref;
+    refs = { ...cellRef, ...refs };
+
+    const sameDate = iterator.getDate() === focusedDate.getDate() && iterator.getMonth() === focusedDate.getMonth();
+    const tabIndex = sameDate ? 0 : null;
+
+    // This needs to occur at the end of this function
+    iterator.setDate(date + 1); return (
+      <td tabIndex={tabIndex} className={className} ref={ref} >
+        {date}
+      </td>
+    );
+  };
+
+  useEffect(() => {
+    const ref = refs[focusedDate.toDateString()];
+    if (ref) {
```

Comment:
I can't test this since the branch isn't deploy but: there is usually some logic about being active here. I think we don't want to focus a date unless the calendar component is active.